src/components/activity.vue

```diff
@@ -25,11 +25,35 @@
         </router-link>
     </div>
   </ion-card-header>
+  
+  <ion-card-header v-else>
+    <div class="avatar-wrap-sml">
+      <router-link v-if="showAuthor" :to="{name: 'ViewUser', params:{userId: author.objectId}}">
+        <avatar :profile="author" size="2em"/>
+      </router-link>
+      <router-link
+        v-else
+        :to="teamLink"
+      >
+          <avatar :profile="team" :name="teamName" v-if="!showAuthor" size="2em"/>
+      </router-link>
+    </div>
+      <div v-if="showAuthor" class="marginRgt">
```

Comment:
Ionic already has content-padding facilities, please use them:

https://ionicframework.com/docs/layout/css-utilities#element-padding

(apply below, too)

```suggestion
      <div v-if="showAuthor" class="ion-padding-end">
```